packages/@uppy/thumbnail-generator/src/index.js

```diff
@@ -302,7 +309,7 @@ module.exports = class ThumbnailGenerator extends Plugin {
     }
   }
 
-  onRestored () {
+  onRestored = () => {
```

Comment:
I'm not familiar with this syntax, will this still be an instance method? If not, what becomes of `this` within the `function`?

Reply:
Yeah, it’s a new thing called “class-properties”: https://robkendal.co.uk/blog/2019-05-13-configure-parcel-js-and-babel-to-use-javascript-proposal-class-properties/, https://babeljs.io/docs/en/babel-plugin-proposal-class-properties. `this` will refer to the class instance in this case, so we don’t have to keep `bind`ing everything.